kodilla-good-patterns/src/main/java/com/kodilla/kodilla/good/patterns/challenges/Separator.java

```diff
@@ -0,0 +1,21 @@
+package com.kodilla.kodilla.good.patterns.challenges;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Separator {
+    public static void main(String[] args) {
+
+        MovieStore movieStore = new MovieStore();
+        List<String> list = new ArrayList<>();
+        movieStore
+                .getMovies()
+                .entrySet()
+                .stream()
+                .map(Map.Entry::getValue)
+                .flatMap(Collection::stream)
+                .map(e->e+"!")
```

Comment:
to wygeneruje ! na koncu, gdzie wykrzyknik mial byc tylko separatorem. Uzyj raczej joining("!")
https://stackoverflow.com/questions/24882927/using-streams-to-convert-a-list-of-objects-into-a-string-obtained-from-the-tostr